Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerTests.cs

```diff
@@ -100,52 +100,76 @@ public IEnumerator TestLinePointers()
             BaseEventSystem.enableDanglingHandlerDiagnostics = true;
         }
 
-        /// <summary>
-        /// Tests that sphere pointer grabs object when hand is insize a giant grabbable
-        /// </summary>
-        [UnityTest]
-        public IEnumerator TestSpherePointerInsideGrabbable()
+
+        private IEnumerator TestPointerViewOfViewHelper(IMixedRealityPointer myPointer, GameObject cube, TestHand testHand)
```

Comment:
Can you rename this? Not understanding viewofview

Reply:
maybe meant FieldOfView?

Reply:
Was supposed to be TestPointerFieldOfViewHelper

Reply:
Was supposed to be TestPointerFieldOfViewHelper